devicetypes/fibargroup/fibaro-double-switch-2-zw5.src/fibaro-double-switch-2-zw5.groovy

```diff
@@ -0,0 +1,639 @@
+/**
+ *  Fibaro Double Switch 2
+ */
+metadata {
+    definition (name: "Fibaro Double Switch 2 ZW5", namespace: "FibarGroup", author: "Fibar Group") {
+        capability "Switch"
+        capability "Energy Meter"
+        capability "Power Meter"
+        capability "Button"
+        capability "Configuration"
+        capability "Health Check"
+
+        command "reset"
+        command "refresh"
+        fingerprint mfr: "010F", prod: "0203", model: "2000"
+        fingerprint deviceId: "0x1001", inClusters:"0x5E,0x86,0x72,0x59,0x73,0x22,0x56,0x32,0x71,0x98,0x7A,0x25,0x5A,0x85,0x70,0x8E,0x60,0x75,0x5B"
+        fingerprint deviceId: "0x1001", inClusters:"0x5E,0x86,0x72,0x59,0x73,0x22,0x56,0x32,0x71,0x7A,0x25,0x5A,0x85,0x70,0x8E,0x60,0x75,0x5B"

```

Comment:
This fingerprint would also match a device with the CCs on the line above. 

Generally, this sort of fingerprinting is intended as a catch-all for devices with generic functionality. A hyper-specific fingerprint like this doesn't serve that utility. I'd be more comfortable just leaving the manufacturer-specific fingerprint.

Reply:
I have made changes just to have the mfr, prod and model number.